aws/resource_aws_lb_target_group_attachment.go

```diff
@@ -2,12 +2,11 @@ package aws
 
 import (
 	"fmt"
-	"log"
-
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/elbv2"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/helper/schema"
+	"log"
```

Comment:
Nit: Please revert this change. The preferred style for Go `imports` is a section of all standard library imports at the top, a new line, and section(s) of third party imports. Each section alphabetized. See also [Go CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#imports) and tooling like `goimports` that will automatically handle this in your editor. 👍